ledger/ledger-utxo/src/main/java/net/corda/v5/ledger/utxo/UtxoLedgerService.java

```diff
@@ -230,6 +230,28 @@ UtxoTransactionBuilder receiveTransactionBuilder(
             @NotNull FlowSession session
     );
 
+    /**
+     * Receives transaction from counterparty.
+     *
+     * @param session The counterparty who are connected to {@link FlowSession}.
```

Comment:
Write this similar to the `finalize` docs to make it clearer, same for `sendTransaction`.